### DISCUSSION TITLE
How to change the border of a window colour ? How to configure the transparency of waybar?

### DISCUSSION THREAD

**Discussion by sanjyay** [prasanthrangan/hyprdots#1501] [Q&A] [answered]
I have two questions 
1)
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/1f0bca9b-5a7a-4ba9-9b89-833fe75f5a78)
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/5b2f4a53-3d93-478f-9fef-21cae7c62bb1)
I want to fix the window border color irrespective of what the wallpapers is
2) 
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/f864989f-4f04-4411-8831-973bf7145baf)
I don't want to have this.I want to just have the modules in waybar. What should I do ?

**Comment by kRHYME7** [comment, marked as answer]
1) If wallbash is enabled/auto, then you can't possibly modify it without changing affecting other wallpaper.
Is you wallbash even set to auto ?   But it looks like you border is not in wallbash to run `  wallbashtoggle.sh -m `  
I forgot the keybind so just search it on your part 
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/15b78e9b-80f7-45cf-8e0a-cbb0dc74a77c)


2) 
`
 layerrule = ignorezero, waybar
`
here's the wiki 
https://wiki.hyprland.org/Configuring/Window-Rules/#rules-1

**Reply by sanjyay** [reply]
I ran wallbashtoggle.sh -m. It was in auto but when i changed it to dark i got this output 
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/88c9e770-9beb-4e75-ba52-86d60c2e7954)
but what should I change wallbash to ?

**Reply by kRHYME7** [reply]
It's working as intended . there's a possibility you have an old ~/.vscode* Directory. Try to purge the unused one. also one of the ~/.config/Code*

should be deleted

**Reply by sanjyay** [reply]
I deleted the old directories and I don't get that error 
1. 
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/14826851-3d18-417d-ba6d-7b9e167059ac)
2.
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/2573f3b3-3cc5-4c0b-93cd-fbfd9421d773)
i want the window border in picture 2 to be like in picture 1.How to do that? . Wallbash in both the pictures are set in dark mode.

**Reply by kRHYME7** [reply]
wallbash colors depend on the wallpaper. It's dynamic

**Reply by sanjyay** [reply]
ohh..thank you
